src/AppWindow.cpp

```diff
@@ -28,55 +29,75 @@ AppWindow::AppWindow(BRect frame)
 	MainView = new AppView(Bounds());
 	AddChild(MainView);
 	
-	PassOut = new RandoTextView(BRect(5,25,200,40), "PassOut",
-		BRect(0,0,200,30), be_bold_font, NULL, B_FOLLOW_LEFT|B_FOLLOW_TOP,
+	// Result Label:
+	BStringView *result = new BStringView(BRect(100,30,11,11),"resultlabel","RESULT");
+	result->ResizeToPreferred();
+	MainView->AddChild(result);
+	
+	// Result:
+	PassOut = new RandoTextView(BRect(5,50,245,80), "PassOut",
+		BRect(5,0,245,60), be_bold_font, NULL, B_FOLLOW_LEFT|B_FOLLOW_TOP,
 		B_WILL_DRAW|B_NAVIGABLE);
 	PassOut->MakeEditable(false);
 	MainView->AddChild(PassOut);
-			
-	ParamsBox = new RandoBox(BRect(5,50,200,146), "ParamsBox", "", B_FOLLOW_LEFT|B_FOLLOW_TOP,
+	
+	// Parameters Label:
+	BStringView *param = new BStringView(BRect(90,90,11,11),"paramlabel","Parameters:");
+	param->ResizeToPreferred();
+	MainView->AddChild(param);
+	
+	// Parameters Box:
+	ParamsBox = new RandoBox(BRect(5,100,245,210), "ParamsBox", "", B_FOLLOW_LEFT|B_FOLLOW_TOP,
```

Comment:
Like @mmuman said, `RandoBox` already has a Label argument, the null string (`""`) after `"ParamsBox"`. However, the layout of the window/view must be changed in order for the label to be visible.